manifests/m/Microsoft/EdgeWebView2Runtime/120.0.2210.121/Microsoft.EdgeWebView2Runtime.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.EdgeWebView2Runtime
+PackageVersion: 120.0.2210.121
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallModes:
+- silentWithProgress
+- silent
+InstallerSwitches:
+  Silent: /silent /install
+  SilentWithProgress: /install
+ProductCode: Microsoft EdgeWebView
+Installers:
+- Architecture: neutral
+  Scope: user
+  InstallerUrl: https://go.microsoft.com/fwlink/p/?LinkId=2124703
+  InstallerSha256: 5B01D964CED28C1FF850B4DE05A71F386ADDD815A30C4A9EE210EF90619DF58E
+- Architecture: neutral
+  Scope: machine
+  InstallerUrl: https://go.microsoft.com/fwlink/p/?LinkId=2124703

```

```suggestion
  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/27b4fd83-937a-4e70-8ee5-c881502ea90e/MicrosoftEdgeWebview2Setup.exe
- Architecture: neutral
  Scope: machine
  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/27b4fd83-937a-4e70-8ee5-c881502ea90e/MicrosoftEdgeWebview2Setup.exe
```
Other PRs have the extracted URLs instead of the compressed link. I searched for `27b4fd83-937a-4e70-8ee5-c881502ea90e` in the repo and didn't find any results, so this might be a unique URL for the version.

---

@stephengillie PR #132219 is the [only one](https://github.com/microsoft/winget-pkgs/tree/master/manifests/m/Microsoft/EdgeWebView2Runtime) whose manifest is still available in the repo. And it uses the same link and installer and was also approved by you. 😅 All prior versions/manifests have been removed in PR #133117 for the reason that the URLs weren't working anymore.
So I suspect that using the extracted URL will only cause it to stop working much sooner than the short link would.
It's a minimal online installer that works for all supported architectures and will always install the latest version of WebView2. So the SHA256 key won't change until this very installer itself is updated. As a result - all I'm doing here is updating the version string to represent the latest version available.
See https://developer.microsoft.com/microsoft-edge/webview2/?form=MA13LH#download for more information. The download link used is the one you get when you click "Get Link" for the Evergreen Bootstrapper.